app/src/main/java/ar/com/wolox/android/example/ui/training/root/RootActivity.java

```diff
@@ -0,0 +1,91 @@
+package ar.com.wolox.android.example.ui.training.root;
+
+import android.content.Intent;
+import android.content.SharedPreferences;
+import android.support.annotation.Nullable;
+import java.util.List;
+import javax.inject.Inject;
+import ar.com.wolox.android.R;
+import ar.com.wolox.android.example.model.User;
+import ar.com.wolox.android.example.network.LoginService;
+import ar.com.wolox.android.example.ui.training.homePage.HomePageActivity;
+import ar.com.wolox.android.example.ui.training.login.LoginActivity;
+import ar.com.wolox.wolmo.core.activity.WolmoActivity;
+import ar.com.wolox.wolmo.networking.retrofit.RetrofitServices;
+import ar.com.wolox.wolmo.networking.retrofit.callback.NetworkCallback;
+import okhttp3.ResponseBody;
+
+/**
+ *
+ */
+public class RootActivity extends WolmoActivity {
+
+    protected String mEmail;
+    protected String mPassword;
+    @Inject
+    protected RetrofitServices mRetrofitServices;
+
+    @Override
+    protected int layout() {
+        return R.layout.activity_base;
+    }
+
+    @Override
+    protected void init() {
+        sharedPreferencesGetInfo();
+        validateLoginToRest(mEmail, mPassword);
+    }
+
+    public void sharedPreferencesGetInfo() {
+        SharedPreferences sharedPreferences = getSharedPreferences(getString(R.string.key_login), 0);
+        String sharedPreferencesEmail = sharedPreferences.getString(getString(R.string.key_email), null);
+        String sharedPreferencesPassword = sharedPreferences.getString(getString(R.string.key_password), null);
+        mEmail = sharedPreferencesEmail;
+        mPassword = sharedPreferencesPassword;
+    }
+
+    /**
+     *
+     * @param email email to validate to Rest
+     * @param password password to validate to Rest
+     */
+    public void validateLoginToRest(String email, String password) {
+
+        mRetrofitServices.getService(LoginService.class).getUserByEmail(email).enqueue(new NetworkCallback<List<User>>() {
+            @Override
+            public void onResponseSuccessful(@Nullable List<User> users) {
+                if (users.size() == 0) {
+                    callLogIn();
```

Comment:
Call this method goToLogin(). The word call could be misunderstood with other functionalities

Reply:
Do the same with the other methods!